labelbox/schema/task.py

```diff
@@ -129,6 +129,16 @@ def failed_data_rows(self) -> Optional[Dict[str, Any]]:
         else:
             return None
 
+    @property
+    def created_data_rows(self) -> Optional[Dict[str, Any]]:
```

Comment:
Just for my information, this seems like a general `Task` class yet here we have methods specific to data row tasks like `failed_data_rows` and `created_data_rows` - a consequence of current task implementation?

Reply:
probably yes, each async task is represented by this Task class I guess, and some have these properties